wimprates/migdal.py

```diff
@@ -70,12 +111,15 @@ def rate_migdal(w, mw, sigma_nucleon, interaction='SI', m_med=float('inf'),
     include_approx_nr = 1 if include_approx_nr else 0
 
     result = 0
+    df_migdal, binding_es_for_migdal = read_migdal_transitions(material=material)
+    if consider_shells is None:
+        consider_shells = _default_shells(material)
     for state, binding_e in binding_es_for_migdal.items():
         binding_e *= nu.eV
         # Only consider n=3 and n=4
```

Comment:
You'll probably want to move this comment up inside _default_shells just before `Xe=['3*', '4*'],` (for other materials you consider different shells)